includes/admin/upgrades/upgrade-functions.php

```diff
@@ -372,6 +372,15 @@ function give_show_upgrade_notices( $give_updates ) {
 			'depend'   => array( 'v20_move_metadata_into_new_table' ),
 		)
 	);
+	
+	// v2.1.5 Add additional capability to the give_manager role.
```

Comment:
@ravinderk I think this update should be related to `v2.2` and not `v2.1.5` as I can see the PR to be assigned to be merged in release `2.2`. Please correct me, if it is intentional.

Reply:
@mehul0810 this code is related to `2.1.5` which appearing here because of merge conflict.